Editor/SettingsAnalysis/PlayerSettingsAnalyzer.cs

```diff
@@ -44,11 +44,42 @@ class PlayerSettingsAnalyzer : ISettingsModuleAnalyzer
             })
         };
 
+        static readonly Descriptor k_IL2CPPCompilerConfigurationMasterDescriptor = new Descriptor(
+            "PAS1004",
+            "Player: IL2CPP Compiler Configuration",
+            new[] { Area.BuildTime },
+            "<b>C++ Compiler Configuration</b> is set to <b>Master</b>. The build time will be longer. Keep this mode only for shipping builds.",
+            "To have optimal build time, change <b>Project Settings ➔ Configuration ➔ C++ Compiler Configuration</b> to <b>Release</b>.")
+        {
+            fixer = (issue => {
+                SetIL2CPPConfigurationToRelease();
+            }),
+
+            messageFormat = "Player : C++ Compiler Configuration is set to 'Master'. The build time will be longer."
```

Comment:
I would omit the " The build time will be longer." as this is already in the description.